src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -122,6 +122,49 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest;
   }
 
+  /**
+   * Function to reimburse a user for their expenses.
+   *
+   * @param recipientId the user who will be receiving the reimbursement
+   * @param amount the amount to be reimbursed
+   * @param submitter the person performing the reimbursement
+   * @returns the created reimbursement
+   */
+  static async reimburseUser(recipientId: number, amount: number, submitter: UserWithTeam): Promise<Reimbursement> {
+    await validateUserIsPartOfFinanceTeam(submitter);
+
+    const totalOwed = await prisma.reimbursement_Request
+      .findMany({
+        where: { recipientId, dateDeleted: null }
+      })
+      .then((userReimbursementRequests: Reimbursement_Request[]) => {
+        return userReimbursementRequests.reduce((acc: number, curr: Reimbursement_Request) => acc + curr.totalCost, 0);
+      });
+
+    const totalReimbursed = await prisma.reimbursement
+      .findMany({
+        where: { purchaserId: recipientId },
```

Comment:
the names of these fields are under Reimbursement in schema.prisma, you have to specify that you're passing recipientId in as purchaserId